src/pages/commits/ActiveCommits/CommitChart.tsx

```diff
@@ -57,3 +95,46 @@ const Bar = styled.div<BarProps>`
   background-color: ${({ color }) => color};
   width: 13px;
 `;
+
+const TooltipContainer = styled.div`
+  width: 150px;
+  height: 160px;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-direction: column;
+  background-color: ${gray.light3};
+`;
+
+const Number = styled(Disclaimer)`
+  width: 40px;
+  font-weight: bold;
+  font-size: 9px;
+  letter-spacing: 0.15px;
+  text-align: center;
+  position: absolute;
+  margin-left: 120px;
+`;
+
+const Circle = styled.div<{ color: string }>`
+  background-color: ${({ color }) => color};
+  border-radius: 4px;
+  width: 8px;
+  height: 8px;
+  margin-right: 18px;
+`;
+
+const TotalCountContainer = styled(Disclaimer)<{ opacity?: number }>`
+  height: 20px;
+  width: 116px;
+  font-size: 9px;
+  letter-spacing: 0.15px;
```

Comment:
Since you're using these text styles in the `Number` component as well, it may be helpful to define a reusable style with the `css` prop. There's a good usage example [here](https://emotion.sh/docs/composition)!

Reply:
Super helpful, thank you!